interop/src/clients/mod.rs

```diff
@@ -0,0 +1,39 @@
+use color_eyre::eyre::Result;
+
+pub mod native;
+pub mod web;
+
+#[derive(Debug)]
+#[non_exhaustive]
```

Comment:
You probably don't need this except if `EmulatedClientType` is used outside this crate
```suggestion
```